src/foam/mlang/sink/Min.js

```diff
@@ -30,6 +30,35 @@ foam.CLASS({
   setValue(getArg1().f(obj));
 }`
     },
+    {
+      name: 'reduce',
+      args: 'foam.mlang.sink.Min sink',
+      type: 'foam.mlang.sink.Min',
+      code: function reduce(sink) {
+        if ( ! sink || ! sink.hasOwnProperty('value') ) return this;
+        if ( ! this.hasOwnProperty('value') ) {
```

Comment:
needed?

Reply:
following the put patterns for min and max now